<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>351</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_name_2">
       <property name="text">
        <string>Ім'я поїзду:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_name_2"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_trainNumber_2">
       <property name="text">
        <string>Номер поїзду:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_trainNumber_2"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_departureTime_2">
       <property name="text">
        <string>Час відправлення:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_departureTime_2"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_departureStation_2">
       <property name="text">
        <string>Станція відправлення:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_departureStation_2"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_route_2">
       <property name="text">
        <string>Маршрут</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_route_2"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_travelDuration_2">
       <property name="text">
        <string>Тривалість подорожі:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEdit_travelDuration_2"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_generalSeats_2">
       <property name="text">
        <string>Загальна кількість місць:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="lineEdit_generalSeats_2"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_coupeSeats_2">
       <property name="text">
        <string>Кількість парних місць:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="lineEdit_coupeSeats_2"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_reservedSeats_2">
       <property name="text">
        <string>Плацкартні місця:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="lineEdit_reservedSeats_2"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_luxurySeats_2">
       <property name="text">
        <string>Люкс місця:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="lineEdit_luxurySeats_2"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_destinationStation_2">
       <property name="text">
        <string>Станція призначення:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_destinationStation_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pbShowTrain">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>420</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Вивести дані</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCreateTrain">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>420</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Створити об'єкт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbQuit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>420</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Вихід</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
